src/pages/LEVANTE/RegisterUsers.vue

```diff
@@ -286,6 +308,9 @@ async function submitUsers() {
   }
 
   try {
+    // showErrorTable.value = true;
+    // throw new Error('test');
```

Comment:
```suggestion
```
Feel free to remove these unless you are leaving them in for debugging or some other purpose.

Reply:
Yea will keep these for now. Still need to add UI feedback for errors.